.github/workflows/build_and_test.yml

```diff
@@ -17,10 +17,54 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  check_changes:
+    name: Check tket library version
+    runs-on: ubuntu-20.04
+    outputs:
+      tket_changed: ${{ steps.filter.outputs.tket }}
+      tket_tests_changed: ${{ steps.filter.outputs.tket_tests }}
+      tket_proptests_changed: ${{ steps.filter.outputs.tket_proptests }}
+      pytket_changed: ${{ steps.filter.outputs.pytket }}
+      tket_ver: ${{ steps.tket_ver.outputs.tket_ver }}
+      tket_package_exists: ${{ steps.tket_package_exists.outputs.tket_package_exists }}
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
+    - uses: dorny/paths-filter@v2
+      id: filter
+      with:
+        base: ${{ github.ref }}
+        filters: |
```

Comment:
What about changes in `https://github.com/CQCL/tket/tree/develop/schemas`

Reply:
Those will have no impact on the tket conan package,

Reply:
Perfect!